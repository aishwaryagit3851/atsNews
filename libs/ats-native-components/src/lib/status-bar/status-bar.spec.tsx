import React from 'react';
import { render } from '@testing-library/react-native';

import StatusBar from './status-bar';

describe('StatusBar', () => {

  it('should should match snapshot', () => {
    const { container } = render(<StatusBar backgroundColor='#fff' />);
    expect(container).toMatchSnapshot();
  });
  it('should render successfully', () => {
    const { container } = render(<StatusBar backgroundColor='#000' />);
    expect(container).toBeTruthy();
  });
});
