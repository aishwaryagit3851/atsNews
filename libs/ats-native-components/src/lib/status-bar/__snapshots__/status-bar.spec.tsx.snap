// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusBar should should match snapshot 1`] = `
ReactTestInstance {
  "_fiber": FiberNode {
    "_debugHookTypes": null,
    "_debugNeedsRemount": false,
    "_debugOwner": null,
    "_debugSource": undefined,
    "actualDuration": 0,
    "actualStartTime": -1,
    "alternate": null,
    "child": FiberNode {
      "_debugHookTypes": null,
      "_debugNeedsRemount": false,
      "_debugOwner": [Circular],
      "_debugSource": undefined,
      "actualDuration": 0,
      "actualStartTime": -1,
      "alternate": null,
      "child": null,
      "childLanes": 0,
      "deletions": null,
      "dependencies": null,
      "elementType": [Function],
      "flags": 5,
      "index": 0,
      "key": null,
      "lanes": 0,
      "memoizedProps": {
        "animated": true,
        "backgroundColor": "#fff",
        "barStyle": undefined,
        "hidden": undefined,
        "translucent": undefined,
      },
      "memoizedState": null,
      "mode": 0,
      "pendingProps": {
        "animated": true,
        "backgroundColor": "#fff",
        "barStyle": undefined,
        "hidden": undefined,
        "translucent": undefined,
      },
      "ref": null,
      "return": [Circular],
      "selfBaseDuration": 0,
      "sibling": null,
      "stateNode": StatusBar {
        "_reactInternalInstance": {},
        "_reactInternals": [Circular],
        "_stackEntry": {
          "backgroundColor": {
            "animated": true,
            "value": "#fff",
          },
          "barStyle": null,
          "hidden": null,
          "networkActivityIndicatorVisible": undefined,
          "translucent": undefined,
        },
        "context": {},
        "props": {
          "animated": true,
          "backgroundColor": "#fff",
          "barStyle": undefined,
          "hidden": undefined,
          "translucent": undefined,
        },
        "refs": {},
        "state": null,
        "updater": {
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
        },
      },
      "subtreeFlags": 0,
      "tag": 1,
      "treeBaseDuration": 0,
      "type": [Function],
      "updateQueue": {
        "baseState": null,
        "effects": null,
        "firstBaseUpdate": null,
        "lastBaseUpdate": null,
        "shared": {
          "interleaved": null,
          "lanes": 0,
          "pending": null,
        },
      },
    },
    "childLanes": 0,
    "deletions": null,
    "dependencies": null,
    "elementType": [Function],
    "flags": 1,
    "index": 0,
    "key": null,
    "lanes": 0,
    "memoizedProps": {
      "backgroundColor": "#fff",
    },
    "memoizedState": null,
    "mode": 0,
    "pendingProps": {
      "backgroundColor": "#fff",
    },
    "ref": null,
    "return": FiberNode {
      "_debugHookTypes": null,
      "_debugNeedsRemount": false,
      "_debugOwner": null,
      "_debugSource": null,
      "actualDuration": 0,
      "actualStartTime": -1,
      "alternate": FiberNode {
        "_debugHookTypes": null,
        "_debugNeedsRemount": false,
        "_debugOwner": null,
        "_debugSource": null,
        "actualDuration": 0,
        "actualStartTime": -1,
        "alternate": [Circular],
        "child": null,
        "childLanes": 0,
        "deletions": null,
        "dependencies": null,
        "elementType": null,
        "flags": 0,
        "index": 0,
        "key": null,
        "lanes": 1,
        "memoizedProps": null,
        "memoizedState": {
          "cache": null,
          "element": null,
          "isDehydrated": false,
          "pendingSuspenseBoundaries": null,
          "transitions": null,
        },
        "mode": 0,
        "pendingProps": null,
        "ref": null,
        "return": null,
        "selfBaseDuration": 0,
        "sibling": null,
        "stateNode": FiberRootNode {
          "_debugRootType": "render()",
          "callbackNode": null,
          "callbackPriority": 0,
          "containerInfo": {
            "children": [],
            "createNodeMock": [Function],
            "tag": "CONTAINER",
          },
          "context": {},
          "current": [Circular],
          "effectDuration": 0,
          "entangledLanes": 0,
          "entanglements": [
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
          ],
          "eventTimes": [
            -1,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
          ],
          "expirationTimes": [
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
          ],
          "expiredLanes": 0,
          "finishedLanes": 0,
          "finishedWork": null,
          "identifierPrefix": "",
          "mutableReadLanes": 0,
          "onRecoverableError": [Function],
          "passiveEffectDuration": 0,
          "pendingChildren": null,
          "pendingContext": null,
          "pendingLanes": 0,
          "pingCache": null,
          "pingedLanes": 0,
          "suspendedLanes": 0,
          "tag": 0,
          "timeoutHandle": -1,
        },
        "subtreeFlags": 0,
        "tag": 3,
        "treeBaseDuration": 0,
        "type": null,
        "updateQueue": {
          "baseState": {
            "cache": null,
            "element": null,
            "isDehydrated": false,
            "pendingSuspenseBoundaries": null,
            "transitions": null,
          },
          "effects": null,
          "firstBaseUpdate": {
            "callback": null,
            "eventTime": 1679655814245,
            "lane": 1,
            "next": null,
            "payload": {
              "element": <StatusBar
                backgroundColor="#fff"
              />,
            },
            "tag": 0,
          },
          "lastBaseUpdate": {
            "callback": null,
            "eventTime": 1679655814245,
            "lane": 1,
            "next": null,
            "payload": {
              "element": <StatusBar
                backgroundColor="#fff"
              />,
            },
            "tag": 0,
          },
          "shared": {
            "interleaved": null,
            "lanes": 0,
            "pending": null,
          },
        },
      },
      "child": [Circular],
      "childLanes": 0,
      "deletions": null,
      "dependencies": null,
      "elementType": null,
      "flags": 1024,
      "index": 0,
      "key": null,
      "lanes": 0,
      "memoizedProps": null,
      "memoizedState": {
        "cache": null,
        "element": <StatusBar
          backgroundColor="#fff"
        />,
        "isDehydrated": false,
        "pendingSuspenseBoundaries": null,
        "transitions": null,
      },
      "mode": 0,
      "pendingProps": null,
      "ref": null,
      "return": null,
      "selfBaseDuration": 0,
      "sibling": null,
      "stateNode": FiberRootNode {
        "_debugRootType": "render()",
        "callbackNode": null,
        "callbackPriority": 0,
        "containerInfo": {
          "children": [],
          "createNodeMock": [Function],
          "tag": "CONTAINER",
        },
        "context": {},
        "current": [Circular],
        "effectDuration": 0,
        "entangledLanes": 0,
        "entanglements": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
        ],
        "eventTimes": [
          -1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
        ],
        "expirationTimes": [
          -1,
          -1,
          -1,
          -1,
          -1,
          -1,
          -1,
          -1,
          -1,
          -1,
          -1,
          -1,
          -1,
          -1,
          -1,
          -1,
          -1,
          -1,
          -1,
          -1,
          -1,
          -1,
          -1,
          -1,
          -1,
          -1,
          -1,
          -1,
          -1,
          -1,
          -1,
        ],
        "expiredLanes": 0,
        "finishedLanes": 0,
        "finishedWork": null,
        "identifierPrefix": "",
        "mutableReadLanes": 0,
        "onRecoverableError": [Function],
        "passiveEffectDuration": 0,
        "pendingChildren": null,
        "pendingContext": null,
        "pendingLanes": 0,
        "pingCache": null,
        "pingedLanes": 0,
        "suspendedLanes": 0,
        "tag": 0,
        "timeoutHandle": -1,
      },
      "subtreeFlags": 7,
      "tag": 3,
      "treeBaseDuration": 0,
      "type": null,
      "updateQueue": {
        "baseState": {
          "cache": null,
          "element": <StatusBar
            backgroundColor="#fff"
          />,
          "isDehydrated": false,
          "pendingSuspenseBoundaries": null,
          "transitions": null,
        },
        "effects": null,
        "firstBaseUpdate": null,
        "lastBaseUpdate": null,
        "shared": {
          "interleaved": null,
          "lanes": 0,
          "pending": null,
        },
      },
    },
    "selfBaseDuration": 0,
    "sibling": null,
    "stateNode": null,
    "subtreeFlags": 5,
    "tag": 0,
    "treeBaseDuration": 0,
    "type": [Function],
    "updateQueue": null,
  },
}
`;
